WordPress/src/main/java/org/wordpress/android/ui/plans/PlansActivity.java

```diff
@@ -98,6 +102,18 @@ public void onClick(View v) {
         } else {
             setupPlansUI();
         }
+
+        // navigate to the "manage plans" page for this blog when the user clicks the manage bar
+        mManageBar.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Blog blog = WordPress.getBlog(mLocalBlogID);
+                if (blog == null) return;
```

Comment:
I know this is just a sanity check, but do you think it'd be better to just navigate to the generic https://wordpress.com/plans/ url if blog is somehow `null` there? I can't imagine how that'd happen though, so it's fine to leave as is if you prefer.